cmd/feedback/feedback_test.go

```diff
@@ -37,3 +38,9 @@ func TestFeedbackCmd_InvalidFlag(t *testing.T) {
 	err := testutils_cobra.ExecutePingcli(t, "feedback", "--invalid")
 	testutils.CheckExpectedError(t, err, &expectedErrorPattern)
 }
+
+// Test Feedback Command with valid profile
+func TestFeedbackCmd_Profile(t *testing.T) {
```

Comment:
Test invalid profile name as well.

Reply:
unable to do this as validation is done before the command is able to finish